package.json

```diff
@@ -114,10 +112,11 @@
 		"@fosscord/util": "dist/util"
 	},
 	"optionalDependencies": {
+		"better-sqlite3": "^8.2.0",
 		"erlpack": "^0.1.4",
 		"nodemailer-mailgun-transport": "^2.1.5",
 		"nodemailer-mailjet-transport": "github:n0script22/nodemailer-mailjet-transport",
 		"nodemailer-sendgrid-transport": "github:Maria-Golomb/nodemailer-sendgrid-transport",
-		"sqlite3": "^5.1.5"
+		"sqlite3": "^5.1.6"
```

Comment:
npm installs optional deps by default. this will install both better-sqlite3 and sqlite3 with the default `npm i`
why not just install `better-sqlite3` by default